From ddba8a08400fb79152fa177e1475288d25137576 Mon Sep 17 00:00:00 2001
From: Simonas Leleiva <simonas.leleiva@jollamobile.com>
Date: Mon, 6 Jan 2014 11:30:06 +0000
Subject: [PATCH 01/10] core: make dependencies compile for 64bit

E.g. pulseaudio-policy-enforcement fails on:
/usr/include/pulsecore/atomic.h:289:2: error: invalid preprocessing directive #warn

Signed-off-by: Simonas Leleiva <simonas.leleiva@jollamobile.com>
---
 src/pulsecore/atomic.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pulsecore/atomic.h b/src/pulsecore/atomic.h
index bca9576..9db01e1 100644
--- a/src/pulsecore/atomic.h
+++ b/src/pulsecore/atomic.h
@@ -286,7 +286,7 @@ static inline int pa_atomic_ptr_cmpxchg(pa_atomic_ptr_t *a, void *old_p, void* n
 
 #elif defined(__GNUC__) && (defined(__amd64__) || defined(__x86_64__))
 
-#warn "The native atomic operations implementation for AMD64 has not been tested thoroughly. libatomic_ops is known to not work properly on AMD64 and your gcc version is too old for the gcc-builtin atomic ops support. You have three options now: test the native atomic operations implementation for AMD64, fix libatomic_ops, or upgrade your GCC."
+#warning "The native atomic operations implementation for AMD64 has not been tested thoroughly. libatomic_ops is known to not work properly on AMD64 and your gcc version is too old for the gcc-builtin atomic ops support. You have three options now: test the native atomic operations implementation for AMD64, fix libatomic_ops, or upgrade your GCC."
 
 /* Adapted from glibc */
 
-- 
1.9.1

